src/libecalc/common/graph.py

```diff
@@ -69,5 +69,8 @@ def get_node(self, node_id: NodeID) -> TNode:
     def sorted_node_ids(self) -> List[NodeID]:
         return list(nx.topological_sort(self.graph))
 
+    def bfs_tree(self, source_id: NodeID) -> List[NodeID]:
```

Comment:
```suggestion
    def breadth_first_search_tree(self, source_id: NodeID) -> List[NodeID]:
```
Please write a small docstring of what this is, and why this is an important way of retrieving data.
I would prefer to have it as a property/method to the network/graph, as a representation in which order it must be evaluated.